assets/src/components/map/tooltip.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import { Tooltip } from "react-leaflet"
+
+interface MapTooltipProps {
+  title: string
+  permanent?: boolean
+}
+
+export const MapTooltip = ({ title, permanent }: MapTooltipProps) => (
+  <Tooltip
+    key={`tooltip-permanent-${permanent}`}
+    className="c-map-tooltip"
+    direction="top"
+    offset={[0, -15]}
+    sticky
+    permanent={permanent}
+  >
+    {title}
+  </Tooltip>
+)
```

Comment:
question: just wondering, do you think the API of this component is better off using a `title` prop instead of `children`? 
Right now, this component ignores that `<Tooltip>` can take a `ReactNode` and not just a `string`, as well as essentially moves `<Tooltip>`'s `children` into the `title` prop.

Reply:
Ooooh that's a good point.

This was a hack because I didn't know the right types to make `children` behave correctly. `PropsWithChildren`.... good to know!

Reply:
and `PropsWithChildren` is just a short cut to get it right "Always™️" (like, in the case they change the type signature or something), but if you look, it's just defined as 

```ts
type PropsWithChildren<P = unknown> = P & { children?: ReactNode | undefined };
```

So you _could_, define `title` like `{ title?: ReactNode | undefined }` if you really preferred to keep `title`

Reply:
https://github.com/mbta/skate/pull/2366/commits/7d8713434b6f83f4aeff7231045929b6de98ca51
https://github.com/mbta/skate/pull/2366/commits/02fecf76217896a4a6483b54ab9bd51f8976bffb